gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -477,7 +477,10 @@ liquibase:
 #    algorithm: None
 #    properties:
 #      rounds: 10
+
 # synchronized roles by event manager to reduce db calls from gateway
 sync:
+  groups:
+    enabled: true
   roles:
-    enabled: false
\ No newline at end of file
+    enabled: true
```

Comment:
**comment**: don't think it should be enabled by default

Reply:
obvious :)